properties/P000025.md

```diff
@@ -1,11 +1,16 @@
 ---
 uid: P000025
-slug: sigma-locally-compact
-name: "$\\sigma$-Locally Compact"
+slug: exhaustible-by-compacts
+name: Exhaustable by compacts
+aliases:
+  - "$\\sigma$-Locally Compact"
 refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
 ---
-A space is $\sigma$-locally compact if and only if it is $\sigma$-compact and locally compact.
+A space that is {P17} and {P23}.
+Equivalently, $X=\bigcup_{n<\omega}K_n$ where for each $x\in X$, there exists
```

Comment:
It should be specified that each $K_n$ is compact.

Reply:
already specified on the next line.